components/skus/browser/rs/lib/src/sdk/orders.rs

```diff
@@ -123,6 +131,15 @@ impl<U> SDK<U>
 where
     U: HTTPClient + StorageClient,
 {
+    #[cfg(feature = "e2e_test")] // just used for testing
```

Comment:
any reason not to use `#[cfg(test)]`?

Reply:
I think we just wanted to scope these just for integration test which are run with the feature e2e_test